### PR TITLE
build(aio): preview server fixes/improvements

### PR DESCRIPTION
**Pull request by gkalpak**
- Minor docs fixes.
- Fix syntax error in Dockerfile.
- Add script for updating the preview server.

### CODE REVIEW

File: aio/aio-builds-setup/docs/vm-setup--update-docker-container.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+
+## Overview
+Assuming you have cloned the repository containing the preview server code (as described
+[here](vm-setup--create-docker-image.md)), you can use the `update-preview-server.sh` script to
+update the preview server based on changes in the source code.
+
+The script will pull the latest changes from the origin's master branch and examine if there have
+been any changes in files inside the preview server source code directory (see below). If there are,
+it will create and new image and verify that is works as expected. Finally, it will stop and remove
```
Comment (petebacondarwin) [inline aio/aio-builds-setup/docs/vm-setup--update-docker-container.md]: create "a" new image

File: aio/aio-builds-setup/docs/vm-setup--update-docker-container.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+
+## Run the script automatically
+You may choose to automatically trigger the script, e.g. using a cronjob. For example, the following
+cronjob entry would run the script every hour and update the preview server (assuming the user has
+the necessary permissions):
+
+```
+# Periodically check for changes and update the preview server (if necessary)
+*/30 *  * * *   /path/to/update-preview-server.sh /path/to/repo /path/to/localcerts /path/to/secrets /path/to/builds /path/to/logs
```
Comment (petebacondarwin) [inline aio/aio-builds-setup/docs/vm-setup--update-docker-container.md]: do you intend to set up such a cronjob?

File: aio/aio-builds-setup/docs/vm-setup--update-docker-container.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# VM setup - Update docker container
+
+
+## Overview
+Assuming you have cloned the repository containing the preview server code (as described
+[here](vm-setup--create-docker-image.md)), you can use the `update-preview-server.sh` script to
+update the preview server based on changes in the source code.
```
Comment (petebacondarwin) [inline aio/aio-builds-setup/docs/vm-setup--update-docker-container.md]: Is it worth pointing out that this script should be run in the VM host? (Is that correct?)

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
One typo!
And a couple of questions.

File: aio/aio-builds-setup/docs/vm-setup--update-docker-container.md (hunk 2 above)
Comment (gkalpak) [inline aio/aio-builds-setup/docs/vm-setup--update-docker-container.md]: Yes :smiley:

File: aio/aio-builds-setup/docs/vm-setup--update-docker-container.md (hunk 3 above)
Comment (gkalpak) [inline aio/aio-builds-setup/docs/vm-setup--update-docker-container.md]: The file's title is "VM setup ...", but I added a note anyway.

**Comment by gkalpak** [comment]
@petebacondarwin, PTAL.

**Review by petebacondarwin — APPROVED** [review]

**Comment by petebacondarwin** [comment]
LGTM - please squash then add the merge label

**Pull request merged**